### PR TITLE
Lesson 1 8

### PR DESCRIPTION
**Pull request by aigerimkwon**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,19 +3,43 @@ import TodoList from "./TodoList";
...
+//const [todoList, setTodoList] = useState(() => {
+//const savedTodoList = JSON.parse(localStorage.getItem("savedTodoList"));
+//return savedTodoList || [];
+//});
+
+//useEffect(() => {
+//ocalStorage.setItem("savedTodoList", JSON.stringify(todoList));
+//}, [todoList]);
+//return [todoList, setTodoList];
+//};
```
Comment (emilford) [inline src/App.js]: You might consider removing these unused lines to keep the code tidy.

File: src/TodoList.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const TodoList = ({ todoList, onRemoveTodo }) => {
             <TodoListItem
               key={todo.id}
               todo={todo}
+              title={todo.title}
```
Comment (emilford) [inline src/TodoList.js:13]: I don't see where you're using `title` in `TodoListItem`.

And looking at the Airtable API [response format](https://airtable.com/developers/web/api/list-records#response) should this be `todo.fields.Title`?

File: src/TodoListItem.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 const TodoListItem = ({ todo, onRemoveTodo, id }) => {
```
Comment (emilford) [inline src/TodoListItem.js:3]: Do you want to destructure `title` here instead of using `todo.fields.Title` below?

**Review by emilford — APPROVED** [review]
A couple of comments and I don't have an API key or Airtable setup to test this with.

Otherwise looking good. 👏

File: src/TodoListItem.js (hunk 3 above)
Comment (aigerimkwon) [reply src/TodoListItem.js:3]: ok! will do!

File: src/TodoList.js (hunk 2 above)
Comment (aigerimkwon) [reply src/TodoList.js:13]: i guess title is not necessary here

File: src/App.js (hunk 1 above)
Comment (aigerimkwon) [reply src/App.js]: Thank you for the quick feedback! The keys are in .env file

**Pull request merged**